src/mlx/warnings_checker.py

```diff
@@ -121,32 +134,63 @@ def return_check_limits(self):
         print("Number of warnings ({0.count}) is between limits {0.warn_min} and {0.warn_max}. Well done.".format(self))
         return 0
 
+    def print_when_verbose(self, message):
+        ''' Prints message only when verbose mode is enabled.
+
+        Args:
+            message (str): Message to conditionally print
+        '''
+        if self.verbose:
+            print(message)
+
 
 class RegexChecker(WarningsChecker):
     name = 'regex'
     pattern = None
 
-    def __init__(self, verbose=False):
```

Comment:
Why was this removed? I think this enabled us to be verbose inside the checkers...